allennlp/tests/semparse/contexts/test_tables/TEST-10.table

```diff
@@ -0,0 +1,56 @@
+row	col	id	content	tokens	lemmaTokens	posTags	nerTags	nerValues	number	date	num2	list	listId
```

Comment:
We typically put files like these in `allennlp/tests/fixtures/data`.  You can see some of them for wikitables here: https://github.com/allenai/allennlp/tree/master/allennlp/tests/fixtures/data/wikitables.  I'd probably add a `mapo_processed` directory there, or something (not really sure how to name it; if you can think of a better name, great).